### ISSUE TITLE
[FEATURE REQUEST] setting the tmux dimension at launch

### ISSUE THREAD

**Issue by TechNZ** [GameServerManagers/LinuxGSM#1396]
currently LGSM has no way to specify the dimensions of new tmux sessions, and as such spawns detached sessions @ 80x23.

at the moment i have only found 2 ways to remedy this, either modifying the command_start.sh file to include -x and -y, this works but is overwritted when when LGSM updates this function.

the other method i found is to create a simple bash script to act as a shim for the tmux command, this works but with some games, permission errors can arise.

im not sure on the exact reason behind the permission problem, however i have only noticed it with arma3 so far, tf2 seems unphased by it.

so long story short, can we have the ability to set the detached sessions width and height at session start, either by a variable in the lgsm script, or even a system wide variable.

**Comment by cedarlug** [comment]
Need more context.

Why is it that your tmux, or lgsm for that matter, NEEDS to be remedied from this default behavior?

**Comment by TechNZ** [comment]
My apologies, i should have been more detailed.
Without setting the size at session creation, there is no way to resize the pane without physically opening and attaching to the pane.

This makes it very annoying for anyone attempting to query the server console using "tmux capture-pane"

In my case this is for a simple web UI, but im sure that other people have other reasons for querying the pane contents.

Please see the attached pictures for comparisons of a default 80x23 session and a 103x23 session
![default_dimensions](https://cloud.githubusercontent.com/assets/2713379/24840336/86009e92-1dbf-11e7-8731-42ab6e642a43.png)
![specific_dimensions](https://cloud.githubusercontent.com/assets/2713379/24840337/86050a22-1dbf-11e7-9ddd-5a48df742ba9.png)

**Comment by TechNZ** [comment]
please ignore, have created a pull request with this functionality.

**Comment by dgibbs64** [comment]
Nice web interface btw. I would be very interested in seeing the code for this :)

**Comment by TechNZ** [comment]
the code is a wee bit messy as i usually code for internal use only, but im sure i could get a git setup for it.

**Comment by dgibbs64** [comment]
Yeah that would be good. It looks interesting and im sure there would be people interested in playing with that. :)

**Comment by TechNZ** [comment]
@dgibbs64 i will be pushing it into https://github.com/TechNZ/LinuxGSM-WebUI after work tonight once i have done a little bit of work removing some of the sensitive data.

**Comment by dgibbs64** [comment]
@TechNZ nice one 👍

**Comment by faflfama** [comment]
Faced some problems @TechNZ ?

**Comment by TechNZ** [comment]
got busy with work and preparing to move house, have tomorrow off so will get the rest of the WebUI up, next project i want to do is a full feature WebUI including installer and basic configuration functionality, but not finding th etime to devote to it at the moment.

**Comment by faflfama** [comment]
Thanks for the info and food luck with the house moving then^^

**Comment by jaredballou** [comment]
Ping me if you want a hand, I'm pretty good at packaging releases :)

**Comment by dgibbs64** [comment]
Any updates on this? Placing in Waiting response.

**Comment by UltimateByte** [comment]
Isn't that already implemented? I remember seing width and so on in tmux start commands.
Maybe we should add the variables to config files now.

**Comment by dgibbs64** [comment]
I think it might be but I cant remember. Not sure if there is a requirement to add variables

**Comment by UltimateByte** [comment]
```
sessionheight
sessionwidth
```

https://github.com/GameServerManagers/LinuxGSM/blob/master/lgsm/functions/command_start.sh#L68-L109
